frontend/src/component/common/AutocompleteBox/AutocompleteBox.tsx

```diff
@@ -59,37 +44,42 @@ export const AutocompleteBox = ({
     value = [],
     onChange,
     disabled,
+    icon,
+    width = '215px',
 }: IAutocompleteBoxProps) => {
-    const [placeHolder, setPlaceholder] = useState('Add Segments');
-    const { classes: styles } = useStyles();
+    const [_, setPlaceholder] = useState('Add Segments');
```

Comment:
This is a little confusing. Why is the default placeholder "add segments"? And if we never use the actual value, why is this line here at all?